torch/csrc/jit/codegen/cuda/scheduler/normalization.cpp

```diff
@@ -13,6 +13,8 @@
 
```

Comment:
This file should be removed from this PR!

Reply:
This is from https://github.com/csarofeen/pytorch/pull/1695 and evidently 1695 damages channels first perf.